macros/external/stage_external_sources.sql

```diff
@@ -71,6 +71,22 @@
 
 {% endmacro %}
 
+{% macro sqlserver__get_external_build_plan(source_node) %}
+
+    {% set build_plan = [] %}
+    
+    {%- set partitions = source_node.external.get('partitions', none) -%}
```

Comment:
This specific line is unnecessary. (It is in the Redshift version, too, just a vestige of a past approach)